dev/ci/ci-basic-overlay.sh

```diff
@@ -128,319 +128,319 @@ project stdpp "https://gitlab.mpi-sws.org/iris/stdpp" ""
+project autosubst 'https://github.com/coq-community/autosubst' '6ba0acccef68c75f6cca8928706c726754d69791'
 # Contact @RalfJung, @co-dan on github
 
-project iris_examples "https://gitlab.mpi-sws.org/iris/examples" "master"
+project iris_examples 'https://gitlab.mpi-sws.org/iris/examples' 'e17e7e92f0a0f52dce65e1189d7c32fa20c23815'
 # Contact @RalfJung, @robbertkrebbers on github
 
 ########################################################################
 # HoTT
 ########################################################################
-project hott "https://github.com/HoTT/HoTT" "master"
+project hott 'https://github.com/HoTT/HoTT' '5661e6850709f6f56e2a80fc3dc2d72498780758'
 # Contact @Alizter, @jdchristensen on github
 
 ########################################################################
 # CoqHammer
 ########################################################################
-project coqhammer "https://github.com/lukaszcz/coqhammer" "master"
+project coqhammer 'https://github.com/lukaszcz/coqhammer' '7dcbc6ad043c4fef109906ad4cbc623c8e343a87'
 # Contact @lukaszcz on github
 
 ########################################################################
 # Flocq
 ########################################################################
-project flocq "https://gitlab.inria.fr/flocq/flocq" "master"
+project flocq 'https://gitlab.inria.fr/flocq/flocq' '561210b49466c1a0f911f6051fa50653e3fc6ca0'
 # Contact @silene on github
 
 ########################################################################
 # coq-performance-tests
 ########################################################################
-project coq_performance_tests "https://github.com/coq-community/coq-performance-tests" "master"
+project coq_performance_tests 'https://github.com/coq-community/coq-performance-tests' 'd1fb22459af24d77b0ab0c224403cfd942f37fe9'
 # Contact @JasonGross on github
 
 ########################################################################
 # coq-tools
 ########################################################################
-project coq_tools "https://github.com/JasonGross/coq-tools" "master"
+project coq_tools 'https://github.com/JasonGross/coq-tools' '1c520114230875e0dbb00216d868391a338febaf'
 # Contact @JasonGross on github
 
 ########################################################################
 # Coquelicot
 ########################################################################
-project coquelicot "https://gitlab.inria.fr/coquelicot/coquelicot" "master"
+project coquelicot 'https://gitlab.inria.fr/coquelicot/coquelicot' 'ca1a747aa8b7ccbfa67a55ae5c8e5c8df71cc396'
 # Contact @silene on github
 
 ########################################################################
 # CompCert
 ########################################################################
-project compcert "https://github.com/AbsInt/CompCert" "master"
+project compcert 'https://github.com/AbsInt/CompCert' '8f5fd33fb4f9594dacbc11597335ec362344c08d'
```

Comment:
As pointed it out by @SkySkimmer, the VST failure might be caused by CompCert, so
```suggestion
project compcert 'https://github.com/AbsInt/CompCert' '2ca39a2801d333abcfa3d691620d03abde4e7e37'
```

Reply:
If it works here, we'll have to do the same on master and ping Andrew.